samples/out/prog17.out

```diff
@@ -0,0 +1 @@
+4
```

Comment:
actually this seems to be the same example that is used in the write-up and this is definitely not the correct output. I think this might be a mistake.

Reply:
Yes, it is incorrect. I believe our interpreter returns the correct output.